### PR TITLE
Expense edit function

### LINKED ISSUE #5: Expense edit function
## Value statement - a requirement, not a solution
As a user
I need an edit function for every expense
so that I can edit the expense when needed

## Description (Text, Scribble, Wireframe, Design)
image

image

## Acceptance criteria
- [ ] the expense detail page holds an edit button
- [ ] on click, the edit button leads to the expense form
- [ ] the expense form is filled with the data of the expense on which i clicked the expense button
- [ ] the data in the form is editable (not the image by now!)
- [ ] after clicking the submit button, i get redirected to the detail view of the expense
- [ ] the detail information of the expense matches with the input data i editet in the form

## Tasks
- [ ] create an edit button on the detail expense page
- [ ] create an edit page for the expenses including the expense form component
- [ ] link the edit button with the edit page
- [ ] edit the expense form, so that i manages the values via multiple states
- [ ] connect the values of the inputs with the states
- [ ] create an additional PUT method in the api route, that manages the edit request
- [ ] connect the submit button with the api route

## Size
medium

### PR DESCRIPTION
**Pull request by edanum**
fixes #5 

Edit function for expenses connected to MongoDB

### CODE REVIEW

File: components/expenses/expenseDetail.js [hunk 1]
```diff
@@ -87,12 +87,14 @@ const Date = styled.p`
   margin: 0px;
 `;
 
-const Description = styled.p`
+const Description = styled.div`
```
Comment (mpagels) [inline components/expenses/expenseDetail.js]: Could this be a `p` tag? (A lot of non-semantic divs here)

File: components/buttons/editButton.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+export default function EditButton() {
+  return (
+    <ToPosition>
+      <Image src={editButton} width={60} height={60} />
+    </ToPosition>
+  );
+}
+
+const ToPosition = styled.div`
```
Comment (mpagels) [inline components/buttons/editButton.js]: Could this be a `button` element instead of a `div`. The component is named `Editbutton`.

**Review by mpagels — APPROVED** [review]
lgtm. Found some semantic improvements.

Good work here.

**Comment by pmarx92** [comment]
LGTM :)

**Pull request merged**